packages/evm/src/evm.ts

```diff
@@ -138,47 +139,54 @@ export class EVM implements EVMInterface {
 
   protected readonly _emit: (topic: string, data: any) => Promise<void>
 
-  private bn128: {
-    ec_pairing: (input_str: string) => string
-    ec_add: (input_str: string) => string
-    ec_mul: (input_hex: string) => string
-  }
+  private _bn128: bn128
 
+  /**
+   * Use this async static constructor for the initialization
+   * of an EVM object
+   *
+   * @param createOpts The EVM options
+   * @returns A new EVM
+   */
   static async create(createOpts?: EVMOpts) {
     const opts = createOpts ?? ({} as EVMOpts)
-    if (opts?.bn128 === undefined) {
-      opts.bn128 = await initRustBN()
-    }
-    return new EVM(opts)
-  }
-  constructor(opts: EVMOpts) {
-    if (opts.bn128 === undefined) {
-      throw new Error('rustbn not provided')
-    }
-    this.bn128 = opts.bn128 // Required to instantiate the EVM
-    this.events = new AsyncEventEmitter()
-
-    this._optsCached = opts
+    const bn128 = await initRustBN()
 
-    this.transientStorage = new TransientStorage()
-
-    if (opts.common) {
-      this.common = opts.common
-    } else {
+    if (opts.common === undefined) {
       const DEFAULT_CHAIN = Chain.Mainnet
```

Comment:
As in `vm`, this feels duplicative.  Can we just pass `Chain.Mainet` to the `common` constructor?